lang/src/org/partiql/lang/ots_work/interfaces/ScalarType.kt

```diff
@@ -1,4 +1,4 @@
-package org.partiql.lang.ots.interfaces
+package org.partiql.lang.ots_work.interfaces
```

Comment:
Below, there are references to `ExprValueType` and `ExprValue`, which are classes in core PartiQL.  These should not be here. 

Either indicate this is a TODO, or we must discuss why not.

Reply:
This is a TODO work. We will decouple scalar run time types from `ExprValueType` later, and migrate them into the plugin interfaces package.